src/components/organisms/ChatSidebar/components/VenueChat/withMessages.ts

```diff
@@ -24,11 +25,13 @@ export const withMessages = <T>(
   const WithMessages = (props: WithMessagesInProps<T>) => {
     const space = props.space;
     const spaceId = space.id;
+    const { world } = useWorldById(props.worldId);
```

Comment:
A more extensible version:

```suggestion
    const worldId = props.worldId;
    const { world } = useWorldById({ worldId });
```

or a step further

```suggestion
    const { world } = useWorldById(props);
```

Reply:
NB